app/assets/javascripts/widgets/Chooser/Chooser.coffee

```diff
@@ -98,16 +100,17 @@ window.Chooser = (config) ->
     setDefault('filterChars', 1)
     setDefault('resetSize', 50)
     setDefault('showNumChosen', true)
-    setDefault('filterText', 'Enter text to narrow search results')
+    setDefault('fromFilterText', 'Enter text to narrow search results')
```

Comment:
I know the text in this box has been this way for a while, so I'm not sure if it needs to change, but I wonder if it would be better if this said something like 'Enter a Short Name to narrow search results' or 'Enter text to narrow search results by Short Name'